.github/workflows/build_test.yml

```diff
@@ -0,0 +1,109 @@
+name: Standard Build and Test
+
+on:
+  # allows us to run workflows manually
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - develop
+      - master
+  push:
+
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+
+jobs:
+  changelog_update:
+    runs-on: ubuntu-latest
+    container:
+      image: alpine:3.14
+
+    name: Is Changelog up-to-date ?
+    steps:
+      - name: Install latest git
+        run: |
+          apk add --no-cache bash git openssh
+          git --version
+
+      - name: Checkout repository
+        uses: actions/checkout@v2
+    
+      - name: Housekeeping
+        run: |
+          cd $GITHUB_WORKSPACE
+          ci/changelog_test.sh
+
+
+  BuildTest:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        pymoab : [
+          _pymoab,
+          ]
+        dagmc : [
+          '',
+          _dagmc
+        ]
+        openmc : [
+          '',
+        ]
+        hdf5 : [
+          '',
+        ]
+        include:
```

Comment:
I'll have to have you explain how this include works relative to the matrix :)

Reply:
the includes add more combinaison to the one defined to the matrix.
for example the `dagmc=''` `pymoab=''` `hdf5=''` `openmc=''` does not exist in the matrix layout (as pymoab is always present)

Reply:
you can use `include` to add more combinaison (or value that don't exist in the matrix) or exclude to `remove` some that exists